src/mscorlib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -8,5 +8,6 @@ internal static partial class Libraries
     {
         internal const string GlobalizationInterop = "System.Globalization.Native";
         internal const string SystemNative = "System.Native";
+        internal const string LibCoreCLR = "libcoreclr";
```

Comment:
We will have to define the PInvoke to libcoreclr somewhere; what is the correct location for this. If we put it in Shared, can we wrap it in #if CORECLR tags?

The fundamental problem is that the coreclr dll exists and can be accessed on CoreCLR, but not on CoreRT. We are punting this problem until later, but we still need CoreRT to build.